src/main/java/com/pikachu/purple/infrastructure/persistence/favorite/entity/FavoriteJpaEntity.java

```diff
@@ -2,22 +2,23 @@
 
 import com.pikachu.purple.domain.favorite.Favorite;
 import com.pikachu.purple.infrastructure.persistence.common.BaseEntity;
-import com.pikachu.purple.infrastructure.persistence.common.EntityStatus;
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
-import jakarta.persistence.EnumType;
-import jakarta.persistence.Enumerated;
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import lombok.AccessLevel;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Getter
 @Entity
 @Table(name = "favorite")
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
+@SQLDelete(sql = "UPDATE favorite SET active = false WHERE favorite_id = ?")
+@Where(clause = "active = true")
```

Comment:
p2) 해당 @Where는 hibernate 6.3부터 deprecated 되었다고 하네요..! 

```suggestion
@SQLRestriction("active = true")
```

관련 [레퍼런스입니다](https://stackoverflow.com/questions/77178963/is-there-a-replacement-for-the-in-6-3-deprecated-where-and-loader)